readme.md

```diff
@@ -10,9 +10,9 @@ It is an approximate string matching library with focus on search and especially
 Can I see a demo ?
 ------------------
 
- You can view the main demo page [here](https://rawgit.com/jeancroy/FuzzySearch/master/demo/autocomplete.html)    
+ You can view the main demo page [here](https://rawgit.com/jeancroy/FuzzySearch/master/demo/autocomplete.html)
```

Comment:
This is my editor removing trailing spaces. I can remove these parts of the changes if this annoys you.